instance/models/openedx_appserver.py

```diff
@@ -176,6 +176,16 @@ class Meta:
             "behind a firewall. (In the django admin, separate group names with a comma.)"
         )
     )
+    additional_monitoring_emails = ArrayField(
```

Comment:
IMHO a Foreign Key / m2m makes sense whenever we're referencing any data that's in the database and an `ArrayField` makes the most sense in other cases for simple string lists (or other lists, since it's a fully-typed list field so you can do lists of integers etc. and the typing is enforced by the DB and converted correctly to python - a big advantage over JSONField). `ArrayField` was probably not available in Django at the time the GitHub admin users list was added.

When we upgrade Postgres we should convert all the JSONFields to official django postgres `jsonb` fields and/or `ArrayField` as appropriate.